### ISSUE TITLE
Sphinx 7.x: ImportError: cannot import name 'setup_js_tag_helper'

### ISSUE THREAD

**Issue by ganto** [ganto/copr-lxc4#31]
Incus currently fails to build for Fedora Rawhide (40) on all architectures:
```
+ sphinx-build -c doc/ -b dirhtml doc/ doc/html/
Running Sphinx v7.2.6

Exception occurred:
  File "/usr/lib/python3.12/site-packages/notfound/extension.py", line 315, in setup
    from sphinx.builders.html import setup_js_tag_helper
ImportError: cannot import name 'setup_js_tag_helper' from 'sphinx.builders.html' (/usr/lib/python3.12/site-packages/sphinx/builders/html/__init__.py)
The full traceback has been saved in /tmp/sphinx-err-ze15izlu.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
Using /builddir/build/BUILD/incus-0.2/_build/bin/incus to generate man pages.
```
I guess there is some breaking change in Sphinx. Fedora 39 still uses Sphinx v6.2.1.

**Comment by ganto** [comment]
Seems to be an issue with the `python3-sphinx-notfound-page` RPM in combination with Sphinx >=7.2 (see readthedocs/sphinx-notfound-page#219). Fixed upstream in >=sphinx-notfound-page-1.0.0. 

Unfortunately Fedora Rawhide still has 0.7.1-9. Fedora [bug 2249216](https://bugzilla.redhat.com/show_bug.cgi?id=2249216).

**Comment by ganto** [comment]
Fixed via https://bodhi.fedoraproject.org/updates/FEDORA-2023-386b6b5bff
